docs/docusaurus/docusaurus.config.js

```diff
@@ -251,12 +251,16 @@ module.exports = {
       '@docusaurus/preset-classic',
       {
         docs: {
-          includeCurrentVersion: false,
+          includeCurrentVersion: true,
```

Comment:
Merged in changes from https://github.com/great-expectations/great_expectations/pull/9400